### PR TITLE
Restore permissions

### PR DESCRIPTION
**Pull request by jlpadilla**
### Related Issue
PR #179 incorrectly removed these permissions.

### Description of changes
- Restore permissions incorrectly removed by PR #179 
- [Permissions deleted](https://github.com/stolostron/search-v2-operator/pull/179/files#diff-b04481aa1896372b1b86820caae41338e60e0ec80beb523a2ab98146ffb5ef23R279-R310)

### CODE REVIEW

File: config/rbac/kustomization.yaml [hunk 1]
```diff
@@ -7,6 +7,8 @@ resources:
 - service_account.yaml
 - role.yaml
 - role_binding.yaml
```
Comment (jlpadilla) [inline config/rbac/kustomization.yaml:9]: These added files, inject the permissions when running `make bundle`.

File: config/rbac/manager_role.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  resources:
+  - configmaps
+  - secrets
+  - serviceaccounts
+  - services
+  - persistentvolumeclaims
+  - pods
+  - endpoints
+  verbs:
+  - '*'
```
[the remark this answers is not in the document]
Comment (jlpadilla) [reply config/rbac/manager_role.yaml]: resolved

**Review by SherinV — APPROVED** [review]
/lgtm
/hold

**Pull request merged**